### PR TITLE
add beta email to pilot

### PR DESCRIPTION
**Pull request by hjanuschka**
adds beta email flag support to pilot - ported from an closed PR https://github.com/fastlane/fastlane/pull/5455

### CODE REVIEW

File: pilot/lib/pilot/build_manager.rb [hunk 1]
```diff
@@ -69,6 +69,11 @@ def distribute(options, build = nil)
         UI.success "Successfully set the changelog and/or description for build"
       end
 
+      if options[:beta_app_feedback_email].to_s.length > 0
+        build.update_build_information!(feedback_email: options[:beta_app_feedback_email])
```
Comment (KrauseFx) [inline pilot/lib/pilot/build_manager.rb]: Shouldn't this also be set on line 68 with the others, so only one request is sent?

**Review by KrauseFx — CHANGES_REQUESTED** [review]

File: pilot/lib/pilot/build_manager.rb (hunk 1 above)
Comment (hjanuschka) [inline pilot/lib/pilot/build_manager.rb]: good point, done.

File: pilot/lib/pilot/options.rb [hunk 2]
```diff
@@ -39,6 +39,11 @@ def self.available_options
                                      optional: true,
                                      env_name: "PILOT_BETA_APP_DESCRIPTION",
                                      description: "Provide the beta app description when uploading a new build"),
+        FastlaneCore::ConfigItem.new(key: :beta_app_feedback_email,
+                                             short_option: "-n",
```
Comment (KrauseFx) [inline pilot/lib/pilot/options.rb]: Could you change the spacing to be the same as the other config items?

**Review by KrauseFx — CHANGES_REQUESTED** [review]

File: pilot/lib/pilot/options.rb (hunk 2 above)
Comment (hjanuschka) [inline pilot/lib/pilot/options.rb]: done

**Review by KrauseFx — APPROVED** [review]

**Comment by KrauseFx** [comment]
Great, thanks for updating the PR 🚀

**Pull request merged**